cabal.project

```diff
@@ -55,95 +55,109 @@ package contra-tracer
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/iohk-monitoring-framework
-  subdir:   contra-tracer
   tag: dd30455144e11efb435619383ba84ce02aee720d
+  --sha256: 1g08bg99fvss99kg27l7pmxm7lh60573xln8l8x2rzvwfvfgk2i5
+  subdir:   contra-tracer
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-prelude
   tag: 00487726c4bc21b4744e59d913334ebfeac7d68e
-  subdir: .
+  --sha256: 0v4fcq5kdd2r5dgwys8kv46ff33qp756n26ycxrca10wq14zkwm5
```

Comment:
Isn't there a script to generate these checksums?

Reply:
> Previously, we had to run `./nix/regenerate.sh` whenever our dependencies changed. What should we do now?

I've restored that script to run cabalProjectRegenerate now. It should be much faster (and less error prone). The only thing to be mindful of is order does matter. That might be improved int he future.

Reply:
The order of the dependencies in `cabal.project`, I assume?

Reply:
No, it's actually the order of sub-nodes of each `source-repository-package`, in particular, `--sha256` must currently appear before `subdir`.